src/ewig/main.cpp

```diff
@@ -121,7 +121,7 @@ void run(int argc, const char** argv, const std::string& fname)
         application{term.size(), key_map_emacs},
         update,
         draw,
-        lager::boost_asio_event_loop{serv, [&] { term.stop(); }},
+        lager::with_boost_asio_event_loop{serv, [&] { term.stop(); }},
```

Comment:
Oh yeah, this is because of changes in Lager (Nix is pinned to an older one) I would have to make sure to upgrade the Nix-provided Lager when merging this.

Reply:
git submodules solves this ;)

Reply:
The way to update the SHA is to try to build and then Nix will complain about the SHA. I can do the update myself, thanks! I used to use `git submodules` but they have their own set of problems so I moved to Nix for this :)

Reply:
Yeah, I know. There is no good globally viable solution at the moment :(